service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -228,19 +228,7 @@ private ApiWorkspaceDescription buildWorkspaceDescription(
             .map(AzureCloudContext::toApi)
             .orElse(null);
 
-    List<ApiTpsPolicyInput> workspacePolicies = null;
-    if (featureConfiguration.isTpsEnabled()) {
-      // New workspaces will always be created with empty policies, but some workspaces predate
-      // policy and so will not have associated PAOs.
-      Optional<ApiTpsPaoGetResult> workspacePao =
-          tpsApiDispatch.getPaoIfExists(
-              new BearerToken(userRequest.getRequiredToken()), workspaceUuid);
-      workspacePolicies =
-          workspacePao
-              .map(ApiTpsPaoGetResult::getEffectiveAttributes)
-              .map(ApiTpsPolicyInputs::getInputs)
-              .orElse(Collections.emptyList());
-    }
+    List<ApiTpsPolicyInput> workspacePolicies = getWorkspacePolicies(userRequest, workspaceUuid);
```

Comment:
optional

To me, this is a little cleaner, WDYT

```
List<ApiTpsPolicyInput> workspacePolicies = featureConfiguration.isTpsEnabled() ? getWorkspacePolicies(userRequest, workspaceUuid) : Collections.emptyList();
```

Reply:
I could go either way. I like the ternary, but the first thing the function does is check isTpsEnabled, so that feels a little funny. I could refactor it a little more but since you marked this as optional I'll just leave it as is.